<launch>
    <!-- Arguments -->
    <arg name="tb0" default="tb3_0" />
    <arg name="tb1" default="tb3_1" />
    <arg name="tb2" default="tb3_2" />

    <include file="$(find multi_nav)/launch/includes/move_base.launch">
        <arg name="tb" value="$(arg tb0)" />
    </include>

    <include file="$(find multi_nav)/launch/includes/move_base.launch">
        <arg name="tb" value="$(arg tb1)" />
    </include>

    <include file="$(find multi_nav)/launch/includes/move_base.launch">
        <arg name="tb" value="$(arg tb2)" />
    </include>
</launch>